ci-operator/step-registry/kueue-operator/jobset-install-operator/kueue-operator-jobset-install-operator-commands.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+echo "Installing Jobset Operator"
```

Comment:
In this case, I think you are fine but there is a precondition for jobset that cert manager must be installed.